src/components/StakingVaults/PositionTable.tsx

```diff
@@ -143,9 +134,14 @@ export const PositionTable: React.FC<PositionTableProps> = ({
         accessor: 'fiatAmount',
         Cell: ({ row }: { row: RowProps }) => {
           // A fiat amount can be positive or negative (debt) but not zero
-          const hasValue = !bnOrZero(row.original.fiatAmount).isZero()
+          const hasValue =
+            bnOrZero(row.original.fiatAmount).gt(0) ||
+            bnOrZero(row.original.fiatRewardsAmount).gt(0)
+
+          const totalFiatAmount = bnOrZero(row.original.fiatAmount).toFixed(2)
+
           return hasValue ? (
-            <Amount.Fiat value={row.original.fiatAmount} />
+            <Amount.Fiat value={totalFiatAmount} />
```

Comment:
ser this PR sucks, crypto amount will have a discrepancy

Reply:
![image](https://github.com/user-attachments/assets/942cf73d-faad-474f-8574-9431fce3c843)

I think it doesn't make sense

Something like that would make sense:
![image](https://github.com/user-attachments/assets/7d75e628-8472-4d29-bce6-e342d0784491)

So if you add every crypto values, you'll end up with the full crypto value of your opportunity

If you add every fiat values, you end up with full fiat value of your opportunity

Another way would be this way:

![image](https://github.com/user-attachments/assets/5b8c7403-2b38-4b76-9161-48f4a2ae2687)

So crypto amount is including the reward amounts, which is equal to the total fiat of the line, and rewards are a separate informations (thought this is less intuitive to me than the other proposed solution)

Reply:
> So if you add every crypto values, you'll end up with the full crypto value of your opportunity

Unfortunately, as mentioned above we can't do that as different denoms exist - you wouldn't be able to add up all crypto amounts, if they're indeed in diff denoms. What if you have multiple rewards entries in various denoms, how would you display that? Say you have a single reward denom for the sake of simplicity, which is different than the staking asset, how would you display that? 

Also, we don't display undelegations currently, so amounts will never fully equal to the total - discussion currently ongoing with product on how to follow-up with this.

Reply:
<img width="1421" alt="image" src="https://github.com/user-attachments/assets/9021950b-5aeb-4951-8978-dc4d36049e37" />

Crypto amount removed from opportunity grand total in 6795920dd7b70828698d97a29e9d3ef18ded80c6

Reply:
Confirmed it's gone, less confusing now 

![image](https://github.com/user-attachments/assets/a058ece9-bcd5-40e1-adfc-2418794b60ad)